src/py42/clients/settings/_converters.py

```diff
@@ -0,0 +1,48 @@
+from py42._compat import str
+
+
+def bool_to_str(value):
+    if isinstance(value, bool) or value in ("true", "false"):
+        return str(value).lower()
+    else:
+        raise ValueError("Value must be True or False")
+
+
+def str_to_bool(value):
+    if value.lower() == "true":
+        return True
+    if value.lower() == "false":
+        return False
+    else:
+        raise ValueError("str_to_bool expects either 'true' or 'false'.")
+
+
+def to_list(value):
+    if isinstance(value, (list, tuple)):
+        return value
+    else:
+        return [value]
+
+
+def to_comma_separated(value):
+    if isinstance(value, (list, tuple)):
+        return ",".join(value)
+    else:
+        return value
+
+
+def comma_separated_to_list(value):
+    return value.split(",")
+
+
+def days_to_minutes(days):
```

Comment:
Is days always an int?

Reply:
Days here will be user supplied when they're setting the attribute, so we don't know. I call `float()` on the input here to be able to handle an input string like `"1.5"`, but also since we want to raise if it's not numeric, the `float()` call will do that for us. 

But I did update `minutes_to_days` to return a float value rather than rounding down, since you _can_ save partial days in the UI.